app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html.twig' %}
```

Comment:
You'll need `{% import 'macros/layout.html.twig' as layout %}` to make the layout helpers work